sockeye/constants.py

```diff
@@ -393,8 +393,22 @@
 DATA_CONFIG = "data.config"
 PREPARED_DATA_VERSION_FILE = "data.version"
 PREPARED_DATA_VERSION = 2
+DEFAULT_FILL_UP_STRATEGY = 'replicate'
 
 # reranking
 RERANK_BLEU = "bleu"
 RERANK_CHRF = "chrf"
 RERANK_METRICS = [RERANK_BLEU, RERANK_CHRF]
+
+# scoring
+SCORING_OUTPUT_ID = 'id'
+SCORING_OUTPUT_SCORE = 'score'
+SCORING_OUTPUT_SOURCE = 'source'
+SCORING_OUTPUT_TARGET = 'target'
+SCORING_OUTPUT_UNKNOWN = '[UNKNOWN_TYPE]'
+SCORING_OUTPUT_DEFAULT = SCORING_OUTPUT_SCORE
+SCORING_OUTPUT_CHOICES = [SCORING_OUTPUT_SCORE, SCORING_OUTPUT_ID, SCORING_OUTPUT_SOURCE, SCORING_OUTPUT_TARGET]
```

Comment:
It would be good to combine them. One option here is that I like the flexibility of having `outputs` take multiple arguments, rather than predefining subsets of all possible outputs (i.e., `--outputs score translation` instead of `--output-format translation_with_score`). That way the user can specify whatever she likes. What do you think?

Reply:
yeah, that'd be nice but its kind of orthogonal to the scoring feature in general. In essence you would only have a single OutputHandler with different configuration depending on the commandline specification of the user. Might make sense to add that in the future, but in a separate PR.

Reply:
Done.